website/docs/d/droplets.html.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: "digitalocean"
+page_title: "DigitalOcean: digitalocean_droplets"
+sidebar_current: "docs-do-datasource-droplets"
+description: |-
+  Retrieve information on Droplets.
+---
+
+# digitalocean_droplets
+
+Get information on Droplets for use in other resources, with the ability to filter and sort the results.
+If no filters are specified, all Droplets will be returned.
+
+This data source is useful if the Droplets in question are not managed by Terraform or you need to
+utilize any of the Droplets' data.
+
+Note: You can use the [`digitalocean_droplet`](/docs/providers/do/d/droplet.html) data source to obtain metadata
+about a single image if you already know the `id`, unique `name`, or unique `tag` to retrieve.
```

Comment:
```suggestion
about a single Droplet if you already know the `id`, unique `name`, or unique `tag` to retrieve.
```

Reply:
Thanks, good catch! Obviously, you can tell what I and Copy McPastington used as a template. :)